### PR TITLE
[ui] Add select components

### PR DESCRIPTION
**Pull request by renestalder**
#84

### CODE REVIEW

**Comment by renestalder** [comment]
See https://socialincome-san.github.io/public/pr-preview/pr-97/?path=/story/components-select-select--standard

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# See https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
+ui/ rene@socialincome.org
+.github/workflows/ui.yml rene@socialincome.org
+.vscode/ rene@socialincome.org
+
+functions/ andras@socialincome.org
```
Comment (andrashee) [inline CODEOWNERS:7]: Should this be admin?

File: ui/src/components/select/Select.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+		},
+		size: {
+			defaultValue: 'base',
+			options: SO_SELECT_SIZES,
+			control: { type: 'select' },
+		},
+	},
+} as ComponentMeta<typeof SoSelect>;
+
+// const Template: ComponentStory<typeof SoSelect> = (args) => <SoSelect {...args} />;
```
Comment (andrashee) [inline ui/src/components/select/Select.stories.tsx:27]: Nit: is the comment required?

**Review by andrashee — APPROVED** [review]
👌

**Review by andrashee — APPROVED** [review]
👌

**Review by andrashee — APPROVED** [review]
👌

File: ui/src/components/select/Select.stories.tsx (hunk 2 above)
Comment (renestalder) [reply ui/src/components/select/Select.stories.tsx]: :heavy_check_mark:

**Pull request merged**